app/src/main/java/edu/grinnell/grinnell_publications_android/Fragments/PublicationsFragment.java

```diff
@@ -39,6 +102,37 @@ public void initializeUI() {
     @Override
     public void onCreateOptionsMenu(Menu menu, MenuInflater inflater) {
         inflater.inflate(R.menu.menu, menu);
+
+        final MenuItem item = menu.findItem(R.id.menu_search);
+        final SearchView searchView = (SearchView) MenuItemCompat.getActionView(item);
+        searchView.setQueryHint("Search by Title");
```

Comment:
Extract this string into a resource. i.e get this string via `Context#getResources(R.string.query_hint)`